eth/db/header.py

```diff
@@ -294,7 +323,7 @@ class AsyncHeaderDB(HeaderDB):
     async def coro_get_canonical_block_hash(self, block_number: BlockNumber) -> Hash32:
         raise NotImplementedError()
 
-    async def coro_persist_header(self, header: BlockHeader) -> Tuple[BlockHeader, ...]:
+    async def coro_persist_header_chain(self, *headers: BlockHeader) -> Tuple[BlockHeader, ...]:
```

Comment:
I think this didn't get updated with the latest:

- lose the `*`
- keep the `coro_persist_header`

Reply:
ah yes, eagle eyes :)